test/unit/test-form.js

```diff
@@ -585,3 +590,101 @@ describes.realWin('isFieldDefault', {}, env => {
     });
   });
 });
+
+describes.fakeWin('isFieldEmpty', {}, env => {
+  let doc;
+
+  beforeEach(() => {
+    doc = env.win.document;
+  });
+
+  describe('text field', () => {
+    let textField;
+
+    beforeEach(() => {
+      textField = doc.createElement('input');
+      textField.type = 'text';
+    });
+
+    it('returns true if the text field is empty', () => {
+      textField.value = '';
+      expect(isFieldEmpty(textField)).to.be.true;
+    });
+
+    it('returns false if the text field is not empty', () => {
+      textField.value = 'some text';
+      expect(isFieldEmpty(textField)).to.be.false;
+    });
+  });
+
+  describe('textarea', () => {
+    let textarea;
+
+    beforeEach(() => {
+      textarea = doc.createElement('textarea');
+    });
+
+    it('returns true if the textarea is empty', () => {
+      textarea.value = '';
+      expect(isFieldEmpty(textarea)).to.be.true;
+    });
+
+    it('returns false if the textarea is not empty', () => {
+      textarea.value = 'some text';
+      expect(isFieldEmpty(textarea)).to.be.false;
+    });
+  });
+
+  describe('checkbox', () => {
+    let checkbox;
+
+    beforeEach(() => {
+      checkbox = doc.createElement('input');
+      checkbox.type = 'checkbox';
+    });
+
+    it('returns false if the checkbox is checked', () => {
+      checkbox.checked = true;
+      expect(isFieldEmpty(checkbox)).to.be.false;
+    });
+
+    it('returns true if the checkbox is not checked', () => {
+      checkbox.checked = false;
+      expect(isFieldEmpty(checkbox)).to.be.true;
+    });
+  });
+
+  describe('radio button', () => {
+    let radio;
+
+    beforeEach(() => {
+      radio = doc.createElement('input');
+      radio.type = 'radio';
+    });
+
+    it('returns false if the radio is checked', () => {
+      radio.checked = true;
+      expect(isFieldEmpty(radio)).to.be.false;
+    });
+
+    it('returns true if the radio is not checked', () => {
+      radio.checked = false;
+      expect(isFieldEmpty(radio)).to.be.true;
+    });
+  });
+
+  describe('dropdown menu', () => {
+    it('always returns false because at least one option is always selected', () => {
+      const dropdown = doc.createElement('select');
+
+      const optionA = doc.createElement('option');
+      dropdown.appendChild(optionA);
+      const optionB = doc.createElement('option');
+      dropdown.appendChild(optionB);
+
+      optionA.selected = false;
+      expect(optionA.selected).to.be.true;
```

Comment:
Why are we asserting this?
This test doesn't need to test the behavior of <select> and <option> - we don't care if optionA or optionB is selected. We only need to test isFieldEmpty always returns false.

So I think what you can do is:
optionA.selected = false;
optionB.selected = false;

expect(isFieldEmpty(dropdown)).to.be.false;

Reply:
The original intention was to show that the option is always selected, even after being set to false. Will make the changes!

Reply:
Yeah that's good. 
If you tend to test that, you can have another test with a proper description. This test, as the name suggests, should focus on if isEmptyField returns false.

If you are going to write that test, I still feel you should set both option to be not selected and assert on one of them being selected anyway. That explains the problem clearly and make optionA and optionB order independent.